deepmd/utils/argcheck.py

```diff
@@ -673,6 +686,13 @@ def descrpt_dpa2_args():
             alias=["repformer_add_type_ebd_to_seq"],
             doc=doc_add_tebd_to_repinit_out,
         ),
+        Argument(
+            "use_econf_tebd",
+            bool,
+            optional=True,
+            default=False,
+            doc=doc_only_pt_supported + doc_use_econf_tebd,
```

Comment:
`use_econf_tebd` for TF is placed in `type_embedding` instead of the descriptor. It's similar as `tebd_dim` for PT and `neuron` in `type_embedding` for TF. 
It's surely because `type_embedding` for TF is outside the descriptor, maybe this way is logically better with a notation here?